### PR TITLE
Stm32mp1 AB fixes

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Mind updating with base/bsp on the commit subject lines?

File: meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc [hunk 1]
```diff
@@ -444,7 +444,7 @@ LINUX_A7_EXAMPLES_DT:remove:stm32mp15-disco = "stm32mp157c-dk2-a7-examples"
 CUBE_M4_EXAMPLES_DT:remove:stm32mp15-disco = "stm32mp157c-dk2-m4-examples"
 UBOOT_DEVICETREE:stm32mp15-disco = "stm32mp157c-dk2"
 SOTA_CLIENT_FEATURES:append:stm32mp15-disco = " ubootenv"
-IMAGE_BOOT_FILES:stm32mp15-disco = "boot.itb"
+IMAGE_BOOT_FILES:stm32mp15-disco = "boot.itb;boot1.itb boot.itb;boot2.itb"
```
[the remark this answers is not in the document]
Comment (igoropaniuk) [reply meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:447]: this is supposed to be so, I just create two different copies of the same file with different names

**Comment by igoropaniuk** [comment]
@ricardosalveti done

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**